packages/actor-init-sparql/lib/ActorInitSparql.ts

```diff
@@ -94,21 +166,18 @@ Options:
     } else if (args.f) {
       query = readFileSync(args.f, { encoding: 'utf8' });
     } else {
-      query = args._.pop();
+      query = args._[args._.length - 1];
```

Comment:
Is there any reason for changing this?

Reply:
Ah, nvm, just saw the commit message.

Reply:
This was because this piece of code used to run after the check if a help message was needed, but now that check is in the buildContext method, this piece of code runs before it, because the buildContext function consumes all the args to make it sources.
If no arguments are given, this index is actually -1, so I will add a check for that.